tools/dev/run_dev_server.sh

```diff
@@ -45,6 +45,10 @@ case $CONFIG in
       ENABLE_MODULES="text2vec-contextionary,backup-filesystem" \
       CLUSTER_GOSSIP_BIND_PORT="7100" \
       CLUSTER_DATA_BIND_PORT="7101" \
+      RAFT_PORT="8300" \
```

Comment:
To help while developing on the branch I made the script parameters always expect 3 node. Happy to change it or make it modular but I think it will help development on the branch to have it hardcoded for now.